pyaedt/workflows/icepak/power_map_from_csv.py

```diff
@@ -0,0 +1,287 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import csv
+import os.path
+
+import pyaedt
+from pyaedt import Icepak
+import pyaedt.workflows
+from pyaedt.workflows.misc import get_aedt_version
+from pyaedt.workflows.misc import get_arguments
+from pyaedt.workflows.misc import get_port
+from pyaedt.workflows.misc import get_process_id
+from pyaedt.workflows.misc import is_student
+
+port = get_port()
+version = get_aedt_version()
+aedt_process_id = get_process_id()
+is_student = is_student()
+
+# Extension batch arguments
+extension_arguments = {"file_path": ""}
+extension_description = "Power map from file"
+
+
+def frontend():  # pragma: no cover
+
+    import tkinter
+    from tkinter import filedialog
+    from tkinter import ttk
+
+    import PIL.Image
+    import PIL.ImageTk
+
+    app = pyaedt.Desktop(
+        new_desktop=False,
+        version=version,
+        port=port,
+        aedt_process_id=aedt_process_id,
+        student_version=is_student,
+    )
+
+    active_project = app.active_project()
+    active_design = app.active_design()
+
+    project_name = active_project.GetName()
+
+    if active_design.GetDesignType() in ["Icepak"]:
```

Comment:
```suggestion
    if active_design.GetDesignType() == "Icepak":
```
Unless other type of designs are expected, we should probably compare with the string directly. If there are other potential values, could you extend the list and discard this suggestion?